robosuite/utils/observables.py

```diff
@@ -376,7 +376,8 @@ def _check_sensor_validity(self):
             _ = self.modality
             self._data_shape = np.array(self._sensor({})).shape
             self._is_number = len(self._data_shape) == 1 and self._data_shape[0] == 1
-        except:
+        except Exception as e:
+            print(f"Error: {e}")
```

Comment:
Can we use the [logging utils](https://github.com/ARISE-Initiative/robosuite/blob/6fbfae0227fbafe90911a9c57a8f9ed4623df6f0/robosuite/utils/log_utils.py) (DefaultLogger) for error messages?